### ISSUE TITLE
Unable to "Reopen in Container", or run any remote container related VSCode extension commands

### ISSUE THREAD

**Issue by scottpageindysoft** [microsoft/vscode-remote-release#4482]
I've been developing in Docker containers for well over 1.5 years, and it stopped working about a month ago.  I've moved on without it for now, and tried again several times this week.  Full uninstall/removal/purge of vscode-insiders (including cache and extension data), full uninstall/install of Docker (same, purging all traces), but still nothing happens when I click "Reopen in container".  I do see in the Log (Extension Host), that the remote container extension failed to load because it's "Missing node-pty", but a search for that results in zero hits.  See step 6 below for log gist.

- VSCode Version:  1.54.0-insider (Universal)
- Local OS Version:  macOS Big Sur 11.2.1
- Remote OS Version:  N/A (unable to open Docker container)
- Remote Extension/Connection Type:  Docker

Steps to Reproduce:

1.  Install vscode-insiders on macOS Big Sur
2. Use existing Docker install or install Docker 
2. Add "Remote Development" extension pack
3. Click "Open a Remote Window" button in lower left corner
4. Click "Reopen in Container"
5. Check OUTPUT -> Log (Extension Host) log
6. Observe the following [error log entries](https://gist.github.com/scottpageindysoft/532451e90791477b9e775ade4802917d)

Does this issue occur when you try this locally?: Yes
Does this issue occur when you try this locally and all extensions are disabled?: Yes

**Comment by chrmarti** [comment]
There are also entries about spdlog not being available. Could this be some security feature, maybe an antivirus, removing native modules?

**Comment by scottpageindysoft** [comment]
Apologies for the late response.
No, I do not have any anti-virus or anti-malware installed on this machine.

I see the spdlog entries, but I don't know why that module isn't installed.  Especially since I've done a clean reinstall of vscode-insiders and the same for all extensions.

I globally installed spdlog, `npm install -g spdlog`, to see if that would help, but the same error shows.
Also, node-pty is showing as missing.

Is there more logging I can perform, or maybe debug the extension?

**Comment by chrmarti** [comment]
What do you get for:
```
ls /Applications/Visual\ Studio\ Code\ -\ Insiders.app/Contents/Resources/app/node_modules.asar.unpacked
```

**Comment by scottpageindysoft** [comment]
[vscode-insiders-ls.txt](https://github.com/microsoft/vscode-remote-release/files/6079638/vscode-insiders-ls.txt)

**Comment by scottpageindysoft** [comment]
Interesting that node-pty is there.
Possibly a path issue?

I recently installed, then removed, Oh My Zsh, and wonder if that messed with something after removing it.
That was about a month or so ago, so it lines up with when I recall noticing this happening.

**Comment by scottpageindysoft** [comment]
I got it working, but I broke my own rule and tried several fixes instead of one at a time.

1.  Installed XCode, as a hunch, because I'm not a native Mac user and XCode always appears to be an issue with something I'm developing elsewhere. (Dev tools, etc).  However, VSCode remote extensions still didn't load without error, and I really didn't think they would.
2. I removed several extensions, many of which I had installed after noticing this issue.  I watched the output log for Log (Extension Host), and any extension that had an error following it, I removed.

The extensions that had errors:  `softwaredotcom.swdc-vscode`, `gruntfuggly.todo-tree`, and unfortunately `eamodio.gitlens-insiders`.

So I installed `eamodio.gitlens` to see if it may have been `eamodio.gitlens-insiders`, and everything appears to be working.
I installed the other extensions that I had removed, and still good-to-go.

Looks like `eamodio.gitlens-insiders` is the culprit.

Before we close this issue, can we get some feedback as to why one extension would bleed its problems onto others?
Maybe I'm missing something, but I don't know what Gitlens would be doing that should interrupt anything the remote containers are.
